transport/http/server.go

```diff
@@ -136,6 +136,12 @@ type ErrorEncoder func(ctx context.Context, err error, w http.ResponseWriter)
 // provided in the context under keys with the ContextKeyResponse prefix.
 type ServerFinalizerFunc func(ctx context.Context, code int, r *http.Request)
 
+// NoOpRequestDecoder is a DecodeRequestFunc that can be used for requests that do not
+// need to be decoded, and simply returns nil, nil.
+func NoOpRequestDecoder(ctx context.Context, r *http.Request) (interface{}, error) {
```

Comment:
I know it might look a bit strange, but I think in Go we usually spell this `NopRequestDecoder`. See for precedent:

https://golang.org/pkg/io/ioutil/#NopCloser
https://godoc.org/github.com/go-kit/kit/log#NewNopLogger
https://godoc.org/golang.org/x/text/transform#NopResetter